src/pevm.rs

```diff
@@ -196,7 +228,8 @@ pub fn execute_revm<S: Storage + Send + Sync>(
     // to avoid "implicit" dependency among consecutive transactions that read & write there.
     // TODO: Refactor, improve speed & error handling.
     let beneficiary_values = mv_memory.consume_beneficiary();
-    Ok(execution_results
+
+    let results = execution_results
```

Comment:
Let's avoid no-ops for now.

Reply:
I find the latter readable but I guess that's subjective.

If we were to use intermediate variables for readability, though, the most valuable addition is the name.
`let results = execution_results.into_iter()...` or
`let execution_results = execution_results.into_iter()...` sound like missed opportunities.

We can settle for `let fully_evaluated_results = ` or `let fully_resolved_results = ` instead.